Servers/controllers/reporting.ctrl.ts

```
@@ -38,6 +40,10 @@ function mapReportTypeToFileSource(
       return "Vendors and risks report";
     case "All reports":
       return "All reports";
+    case "Annexes report":
+      return "Annexes report";
+    case "All reports":
```

Biome (1.9.4)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)